### PR TITLE
[IF-INTERLAY2-06], [IF-INTERLAY2-07] Minor suggestions on Issue, Redeem, and Replace specs

### PR DESCRIPTION
**Pull request by shonfeder**
This PR suggests a few very minor fixes/alterations surfaced from @informalsystems' audit of interlay/btc-parachain at
75fd3e2

The only suggestion that has any substance is that in 655f606, please see that commit for details.

### CODE REVIEW

**Review by gregdhill — APPROVED** [review]

File: interbtc-spec/docs/source/spec/issue.rst [hunk 1]
```diff
@@ -70,12 +70,12 @@ Maps
...
 
-Users create issue requests to issue interBTC. This mapping provides access from a unique hash ``IssueId`` to a ``Issue`` struct. ``<IssueId, Issue>``.
+Users create issue requests to issue interBTC. This mapping provides access from a unique hash ``IssueId`` to a ``Issue`` struct. ``<IssueId, IssueRequest>``.
 
 Structs
 -------
 
-Issue
+IssueRequest
 .....
```
Comment (gregdhill) [inline interbtc-spec/docs/source/spec/issue.rst:79]:
```suggestion
IssueRequest
............
```

File: interbtc-spec/docs/source/spec/redeem.rst [hunk 2]
```diff
@@ -86,13 +86,13 @@ Maps
...
-Users create redeem requests to receive BTC in return for interBTC. This mapping provides access from a unique hash ``redeemId`` to a ``Redeem`` struct. ``<redeemId, Redeem>``.
+Users create redeem requests to receive BTC in return for interBTC. This mapping provides access from a unique hash ``redeemId`` to a ``Redeem`` struct. ``<redeemId, RedeemRequest>``.
 
 
 Structs
 -------
 
-Redeem
+RedeemRequest
 ......
```
Comment (gregdhill) [inline interbtc-spec/docs/source/spec/redeem.rst:96]:
```suggestion
RedeemRequest
.............
```

**Review by gregdhill — COMMENTED** [review]
Thanks for the update @shonfeder! Just a couple small formatting issues, but otherwise LGTM!

**Comment by shonfeder** [comment]
Fixed. Thanks!

**Pull request merged**